### PR TITLE
Different Types of Cards Made

### PR DESCRIPTION
**Pull request by Asypher00**
Different card variants as stated in LMS
Screenshot (6)
S

### CODE REVIEW

File: Card.css [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+}
+.txt-style{
+  color : rgb(0, 0, 0);
+  text-decoration: none;
+}
+
+.card-badge{
+  position: absolute;
+  text-align: center;
+  background-color: rgb(96, 248, 96);
```
Comment (Sameony) [inline Card.css:69]: Prefer declaring custom variables for colors, as it is difficult to recognize them with such codes.

File: Card.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>replit</title>
```
Comment (Sameony) [inline Card.html]: the title is usually related to the content of the webpage. Ignore this if its related to the content.

File: Card.html [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+                      <!-- <img class="card-img"src="/images/nike.jpg" alt=""> -->
+                      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Aut maiores officiis fugiat tempore laborum soluta quia, et aperiam excepturi harum, facilis corrupti necessitatibus recusandae repellendus! Nihil doloremque sunt alias magni?</p>
+                      <!-- <div class="card-main">
+                         <p class="prod-name">Nike Air Jordans 1</p>
+                         <p class="prod-type">Men Shoes Sneakers</p>
+                         <p class="prod-price">Rs. 30,000</p>
+                      </div>
+                         <button class="btn txt-style">BUY NOW</button>
+                         <button class="btn btn-icon"><i class="fas fa-shopping-cart fa-2x"></i></button>
+                         <div class="card-badge">4.5<i class="fas fa-star"></i> </div>-->
```
Comment (Sameony) [inline Card.html:99]: Remove Unnecessary comments

File: Card.css [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+  display: flex;
+  gap : 1.25rem;
+  flex-wrap: wrap;
+  
+}
+
+.card-container {position: relative;
+  display : flex;
+  
+  /* flex-wrap : wrap; */
```
Comment (shuntzhere) [inline Card.css]: You may remove the commented out code and format it using an extension like prettier.

File: Card.css [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+  flex-direction : column;
+  justify-content: space-between;
+  background-color: lightgray;
+  border : 0.125rem solid black;
```
Comment (shuntzhere) [inline Card.css:10]: Don't use fixed colors, save the color in some variable and then use them across your library.

File: Card.html [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+           <p class="prod-type">Men Shoes Sneakers</p>
+           <p class="prod-price">Rs. 30,000</p>
+        </div>
+           <button class="btn txt-style">BUY NOW</button>
+           <button class="btn btn-icon"><i class="fas fa-shopping-cart fa-2x"></i></button>
+           <div class="card-badge">4.5<i class="fas fa-star"></i> </div>
+           <button class="btn btn-icon pos-top-right"><i class="fas fa-times-circle fa-2x"></i></button>
+           </div>
+
+         <h1></h1>
```
Comment (shuntzhere) [inline Card.html:56]: These tags may be removed as this is not doing anything.

File: Card.html [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
```
Comment (shuntzhere) [inline Card.html:71]: Same here, this empty tag can be removed too.

File: Card.html [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+                  <p class="prod-price">Rs. 30,000</p>
+               </div>
+                  <button class="btn txt-style">BUY NOW</button>
+                  <button class="btn btn-icon"><i class="fas fa-shopping-cart fa-2x"></i></button>
+                  <div class="card-badge">4.5<i class="fas fa-star"></i> </div>
+                  <button class="btn btn-icon pos-top-right"><i class="fas fa-times-circle fa-2x"></i></button>
+                  <span class="txt-overlay">Hurry Up, Only 10 Left in Stock</span>
+                  </div>
+
+                  <h1></h1>
```
Comment (shuntzhere) [inline Card.html:87]: Indent the code so that it is more readable. And another h1 tag is here which can be removed.

**Review by shuntzhere — CHANGES_REQUESTED** [review]
Apply Indentation and don't leave commented out code .

**Pull request merged**